Makefile

```diff
@@ -44,11 +45,11 @@ $(BUILDDIR)/%.o: src/%.c
 
 $(BUILDDIR)/test_%: test/%.c src/%.c
 	@echo "BUILDING $@"
-	$(MUTE)$(CC) -o $@ -Isrc -W -Wall $(CFLAGS) $(LDFLAGS) -std=gnu11 -lcmocka $^
+	$(MUTE)$(CC) -o $@ -Isrc -g $(TFLAGS) $(LDFLAGS) -std=gnu11 -lcmocka $^
```

Comment:
Initially I hardcoded `-g` there, and then I decided that supplying custom cflags to tests makes sense, so I added TFLAGS. I just forgot to remove that hardcoded `-g`.